src/cljs/harja/views/urakka/yllapitokohteet/paikkaukset/paikkaukset_toteumalomake.cljs

```diff
@@ -463,7 +463,8 @@
                  maara))))
 
 (defn- footer-vasemmat-napit [e! toteumalomake muokkaustila?]
-  (let [voi-tallentaa? (::tila/validi? toteumalomake)]
+  ;; Poista tallennus käytöstä kun paikkauskohde on valmis
+  (let [voi-tallentaa? (and (::tila/validi? toteumalomake) (not (= "valmis" (:paikkauskohde-tila toteumalomake))))]
```

Comment:
(not= "valmis" (:paikkauskohde-tila toteumalomake))
Olisi lyhyempi muoto kuin :)
(not (= "valmis" (:paikkauskohde-tila toteumalomake)))